corehq/apps/commtrack/static/commtrack/js/products_and_programs.async.js

```diff
@@ -98,3 +92,12 @@ ko.bindingHandlers.isPaginationActive = {
         }
     }
 };
+
+$(function(){
```

Comment:
@dimagi/js-team One of the common use cases of inline js is to initialize complex widgets. It could be interesting to use knockout components more widely, but for now I'm just giving knockout widgets a class name prefixed with `ko-` and initializing them manually because it's less of a change.